### ISSUE TITLE
Not working with Max-height 

### ISSUE THREAD

**Issue by Northout-Salman** [jkuri/ngx-slimscroll#89]
Hi, 
i have added this plugin for max-height but its not working, working fine with height.
here is code
`ng-scrollbar #scrollEl class="tooltip_scroll">
									    	<a class="dropdown-item" href="#" title="House of Blues">
									    	   <span class="limit-textoverflow">House of Blues</span>
  									        </a>
  									        <a class="dropdown-item" href="#" title="House of Blues">
									    	   <span class="limit-textoverflow">House of Blues</span>
  									        </a>
  									        <a class="dropdown-item" href="#" title="House of Blues">
									    	   <span class="limit-textoverflow">House of Blues</span>
  									        </a>
  									        <a class="dropdown-item" href="#" title="House of Blues">
									    	   <span class="limit-textoverflow">House of Blues</span>
  									        </a>
									     </ng-scrollbar>`
`.tooltip_scroll {
	max-height:70px;
}`

basically i want to show scroll when there is more items.

**Comment by jkuri** [comment]
setting height is mandatory for this module to work.

**Comment by Northout-Salman** [comment]
Its working fine with height. What i need is it should work only with **Max-height**

**Comment by jkuri** [comment]
I think I have been clear this cannot work only with max-height, height is mandatory.

**Comment by Northout-Salman** [comment]
Ok. Thank you very much.
